app/src/pages/Lobby/index.tsx

```diff
@@ -125,19 +127,32 @@ function SettingsSection() {
               )}
             </Grid>
             <Grid item>
-              <SecondsPerTurnInput
-                value={String(currentGame.seconds_per_turn || "")}
-              ></SecondsPerTurnInput>
+              <Typography variant="h6" className={titleClasses.title}>
+                Cards
+              </Typography>
```

Comment:
Haha we've been back and forth on this - I'm pro it.  And the more we add here, the more justified it becomes

cc: @wtachau

Reply:
I wonder if we could use some icon, or emoji here to distinguish the headers. they blend in a bit rn